src/file_import_retriever.rs

```diff
@@ -0,0 +1,162 @@
+use std::collections::{HashMap, HashSet};
```

Comment:
Since this file's main export is 2 functions which are verbs (get_imports_from_file, get_imports_map), it should be named a verb as well, maybe something like `get_imports` rather than `file_import_retriever`.

`good-fences_runner` is a noun because its main export is a struct + that struct's impl.